File: Be/property-service/src/main/java/com/property/dto/response/SummaryResponse.java
package com.property.dto.response;

import java.math.BigDecimal;
import java.util.Set;

import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

@Setter
@Getter
@Builder
public class SummaryResponse {

    long totalApartments;
    long totalRoomTypes;
    long totalRooms;
    long totalTenants;
    BigDecimal totalIncome;

//    Set<TotalInvoiceResponse> totalInvoiceResponses;
}

File: Fe/client/src/dto/response/SummaryResponse.ts
export type SummaryResponse = {
	totalApartments: number
	totalRoomTypes: number
	totalRooms: number
	totalTenants: number
	totalIncome: number
};

export type totalInvoiceResponse = {
	total: number
	month: number
	year: number
};

export type invoiceUnPaidResponse = {
	totalUnpaidAmount: number
	totalUnpaidInvoices: number
}
